package ex_240304;

public class Ex_01_try_catch_1 {

	public static void main(String[] args) {
		// try catch 연습장
		// 비정상적인 종료 막기 위해서, 미리 예외로 처리를 한다.
		// 알고 대비를 해서, 예측하지 못한 비정상적인 종료를 막을수 있다.
		// 만약, 예외 처리를 하지 않으면, 해당 프로그램이 정상 종료가 아니라,
		// 비정상이라서, 동작을 정상 수행을 못하게됨.
		// 예) 파일을 입출력, 네크워크 통신을 통해서, 데이터를 못받아올때
		// 예외 처리하는 형식도 각각의 디테일하게 조정을 할수 있지만
		// 그 디테일한 예외처리를 포함한, 범위가 넒은 예외를 처리하는 경우도 있음.

	}

}
